contrib/PyCuAmpcor/src/cuEstimateStats.cu

```diff
@@ -113,18 +111,19 @@ __global__ void cudaKernel_estimateVar(const float* corrBatchRaw, const int NX,
 
```

Comment:
Line 102/100, for small peak values, make it 99, 99, 0., i.e., xy=0.

Reply:
Done.